[32mINFO    [0m logs:base_api.py:125 api变化的模板参数是:{
  "token": "FTsumt3j6txqKydN103S6fHcMRa4JADvDUohu_kLMIzcLDopz407TMV-D1f9Vj8GPd1NSgneHDGFv_tbyxxKusx6qmYWXogj2iseIpbpq7ukAMNCqnUjFhROLqMnRmDQBUwCzlam62jzXrWnMQG-I_aWRFCOy79g_Smh1ZTgTlXpV7HGYMvP2SzDoHepdMTb1pygm58991iWJdFT007M3w",
  "name": "name",
  "userid": "edit",
  "mobile": "15172222165"
}
[32mINFO    [0m logs:base_api.py:135 请求修改后的数据是：{
  "json": {
    "department": [
      1,
      3
    ],
    "mobile": 15172222165,
    "name": "name",
    "userid": "edit"
  },
  "method": "post",
  "params": "access_token=FTsumt3j6txqKydN103S6fHcMRa4JADvDUohu_kLMIzcLDopz407TMV-D1f9Vj8GPd1NSgneHDGFv_tbyxxKusx6qmYWXogj2iseIpbpq7ukAMNCqnUjFhROLqMnRmDQBUwCzlam62jzXrWnMQG-I_aWRFCOy79g_Smh1ZTgTlXpV7HGYMvP2SzDoHepdMTb1pygm58991iWJdFT007M3w",
  "url": "https://qyapi.weixin.qq.com/cgi-bin/user/create"
}
[32mINFO    [0m logs:base_api.py:137 返回的数据为：{
  "errcode": 60102,
  "errmsg": "Warning: wrong json format. userid existed, hint: [1634477114_161_b8a719a3eb9161131a931d51824c1610], from ip: 222.212.179.224, more info at https://open.work.weixin.qq.com/devtool/query?e=60102"
}
[32mINFO    [0m logs:test_member.py:58 ------开始测试修改成员userid存在，name不为空，电话不重复，断言成功------
[32mINFO    [0m logs:base_api.py:125 api变化的模板参数是:{
  "token": "FTsumt3j6txqKydN103S6fHcMRa4JADvDUohu_kLMIzcLDopz407TMV-D1f9Vj8GPd1NSgneHDGFv_tbyxxKusx6qmYWXogj2iseIpbpq7ukAMNCqnUjFhROLqMnRmDQBUwCzlam62jzXrWnMQG-I_aWRFCOy79g_Smh1ZTgTlXpV7HGYMvP2SzDoHepdMTb1pygm58991iWJdFT007M3w",
  "name": "edit",
  "userid": "edit",
  "mobile": "15172999999"
}
[32mINFO    [0m logs:base_api.py:135 请求修改后的数据是：{
  "json": {
    "department": [
      1,
      3
    ],
    "mobile": 15172999999,
    "name": "edit",
    "userid": "edit"
  },
  "method": "post",
  "params": "access_token=FTsumt3j6txqKydN103S6fHcMRa4JADvDUohu_kLMIzcLDopz407TMV-D1f9Vj8GPd1NSgneHDGFv_tbyxxKusx6qmYWXogj2iseIpbpq7ukAMNCqnUjFhROLqMnRmDQBUwCzlam62jzXrWnMQG-I_aWRFCOy79g_Smh1ZTgTlXpV7HGYMvP2SzDoHepdMTb1pygm58991iWJdFT007M3w",
  "url": "https://qyapi.weixin.qq.com/cgi-bin/user/update"
}
[32mINFO    [0m logs:base_api.py:137 返回的数据为：{
  "errcode": 0,
  "errmsg": "updated. Warning: wrong json format. "
}
[32mINFO    [0m logs:test_member.py:60 ------打印的相应结果：{'errcode': 0, 'errmsg': 'updated. Warning: wrong json format. '}------
[32mINFO    [0m logs:test_member.py:61 ------修改成员测试结束------